code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -46,6 +46,7 @@
 	equip_delay_other = 80
 	resistance_flags = NONE
 	actions_types = list(/datum/action/item_action/toggle_spacesuit)
+	equip_time = 1.5 SECONDS
```

Comment:
I know we just had a conversation about equipping suits in space being a problem while moving, but actually I think space suits taking even longer than 1.5 seconds is actually probably a better representation of their bulk and cumbersome nature, and really makes the suits that don't have slowdown significantly more desirable and premium for their utility.

I don't think hardsuits should be things you just hotswap into easily, as an example.

Reply:
increased to 4.5 seconds

Reply:
this kills tactical space battery swapping

Reply:
the less we talk about spacesuit batteries the better.